packages/pigment-css-unplugin/src/index.ts

```diff
@@ -115,7 +115,7 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
     ...rest
   } = options;
   const finalTransformLibraries = transformLibraries
-    .concat(process.env.RUNTIME_PACKAGE_NAME as string)
+    .concat([process.env.RUNTIME_PACKAGE_NAME as string, '@mui/material-pigment-css'])
```

Comment:
Is this a quick solution for `@mui/material-pigment-css`? Because if someone wants to build an adapter on top of Pigment CSS, they have to add it to the list here.

Reply:
Nothing to change at this point but it's something to think about for Pigment CSS v1.

Reply:
It's not only for material-pigment-css.
It's for any other similar libraries. They'll just have to ask users to add their lib to the transformLibraries list.

Reply:
Then why do we need to add `@mui/material-pigment-css` here? It should be inside a config `@mui/material-pigment-css`.

Something like:

```js
import { withPigment } from '@pigment-css/nextjs-plugin';
import { presets } from '@mui/material-pigment-css';

withPigment({
  theme,
  …presets, // transformLibraries: ['@mui/material-pigment-css']
})
```

Reply:
We don't need any config in `@mui/material-pigment-css` itself. There are two ways to handle this library through Pigment -

1. In the Material docs for configuring Pigment, ask users to add `@mui/material-pigment-css` to the `transformLibraries` option. This would also be applicable for other 3rd party libraries built on Pigment.
2. Or, make this option built-in since we control both the libraries.

Reply:
There is an `overrideContext` that we might need to add as a workaround for the `toString` issue.

This is the config that works for Material UI x Pigment:

```js
export default withPigment(nextConfig, {
  theme: customTheme,
  transformLibraries: ["@mui/material"],
  overrideContext: (context) => {
    if (!context.$RefreshSig$) {
      context.$RefreshSig$ = outerNoop;
    }
    return {
      ...context,
      require: (id) => {
        if (id === "@mui/styled-engine" || id === "@mui/styled-engine-sc") {
          return {
            __esModule: true,
            default: () => () => () => null,
            internal_processStyles: () => {},
            keyframes: () => "",
            css: () => "",
          };
        }
        return context.require(id);
      },
    };
  },
});
```

Where do you think the `overrideContext` should be?